src/Stratis.Bitcoin.Features.PoA/PoAMiner.cs

```diff
@@ -188,6 +188,17 @@ private void GatherMiningStatistics()
             log.AppendLine(">> Miner");
             log.AppendLine();
 
+            if (!this.federationManager.IsFederationMember)
+            {
+                log.AppendLine("Mining information is not available for non federation members.");
+                log.AppendLine("It is possible that your node was kicked from the federation due to inactivity.");
```

Comment:
This can be added subject to their being a pubkey for this node. One could even check the polls in order to show a more relevant message.

Reply:
Thanks Gustav, will add to next PR.